src/ui/components/DefinitionList/index.js

```diff
@@ -0,0 +1,50 @@
+/* @flow */
+import * as React from 'react';
+import makeClassName from 'classnames';
+
+import './styles.scss';
+
+
+type DefinitionItemProps = {|
+  children: React.Node,
+  className?: string,
+  title: React.Node,
```

Comment:
No biggie but `term` might be a better name to illustrate how it's just a wrapper for `<dt>{term}</dt>`

Reply:
Good call 👍